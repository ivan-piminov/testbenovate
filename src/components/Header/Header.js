import React from "react";
import {MAIN, TABLES} from "../Routes/Routes";
import {NavLink} from "react-router-dom";
import Button from '@material-ui/core/Button';
import Box from "@material-ui/core/Box";
import {makeStyles} from "@material-ui/core/styles";

// "шапка" приложения с 2 кнопками навигации
export function Header() {
    const useStyles = makeStyles({
        header: {
            textDecoration: "none"
        }
    });

    const classes = useStyles();

    return (
        <div>
            <Box display="flex" p={2} bgcolor="#4791db" justifyContent="space-around">
                <NavLink to={MAIN} className={classes.header}> <Button variant="contained"> Главная </Button></NavLink>
                <NavLink to={TABLES} className={classes.header}><Button variant="contained"> Таблицы </Button></NavLink>
            </Box>
        </div>
    );
}