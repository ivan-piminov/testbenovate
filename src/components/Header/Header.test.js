import React from "react";
import {Header} from './Header';
import {shallow} from "enzyme";

describe('Testing <Header/>',()=>{
    it('Header render correctly', ()=>{
        const header = shallow(<Header/>);
        expect(header).toMatchSnapshot();
    })
});