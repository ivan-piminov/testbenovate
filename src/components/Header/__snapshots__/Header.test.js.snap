// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <Header/> Header render correctly 1`] = `ShallowWrapper {}`;
